python/sample-apps/function/lambda_function.py

```diff
@@ -1,26 +1,14 @@
 import os
 import json
-import aiohttp
-import asyncio
 import boto3
-
-
-async def fetch(session, url):
-    async with session.get(url) as response:
-        return await response.text()
-
-
-async def callAioHttp():
-    async with aiohttp.ClientSession() as session:
-        html = await fetch(session, "http://httpbin.org/")
+import requests
 
 s3 = boto3.resource("s3")
 
 # lambda function
 def lambda_handler(event, context):
 
-    loop = asyncio.get_event_loop()
-    loop.run_until_complete(callAioHttp())
+    requests.get("https://www.amazon.com/")
```

Comment:
This should probably remain `httpbin.org`.

Reply:
Done